wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -157,6 +168,7 @@ async def get_status(self) -> Status:
         status = LaunchKubernetesMonitor.get_status(self.name)
         if status in ["stopped", "failed", "finished", "preempted"]:
             await self._delete_secret()
+            await self._delete_auxiliary_resources_by_label()
```

Comment:
I've never noticed this before, but this is really gross, deleting stuff on a call to get_status. Out of scope for this PR, but we should fix that. Filing a ticket